libraries/botframework-streaming/src/assemblers/payloadAssembler.ts

```diff
@@ -8,32 +8,36 @@
 import { SubscribableStream } from '../subscribableStream';
 import { StreamManager, PayloadTypes } from '../payloads';
 import { ContentStream } from '../contentStream';
-import { IAssemblerParams } from '../interfaces/IAssemblerParams';
-import { IHeader } from '../interfaces/IHeader';
-import { IResponsePayload } from '../interfaces/IResponsePayload';
-import { IReceiveResponse, IReceiveRequest } from '../interfaces';
-import { IRequestPayload } from '../interfaces/IRequestPayload';
+import { IHeader, IResponsePayload, IReceiveResponse, IReceiveRequest, IRequestPayload } from '../interfaces';
+
+interface AssemblerParams {

```

Comment:
What's the reasoning behind introducing this over `IAssemblerParams`?

Reply:
I don't remember my rationale... I've removed `AssemblerParams` and applied its stricter `onCompleted` type to `IAssemblerParams.onCompleted`. 😐